cfme/utils/ftp.py

```diff
@@ -334,9 +334,13 @@ def _callback(line):
             fields = re.split(r"\s+", line, maxsplit=8)
             # This is because how informations in LIST are presented
             # Nov 11 12:34 filename (from the end)
-            date = strptime(str(datetime.now().year) + " " + fields[-4] + " " + fields[-3] + " " +
-                            fields[-2],
-                            "%Y %b %d %H:%M")
+            try:
+                date = strptime(
+                    f"{datetime.now().year} {fields[-4]} {fields[-3]} {fields[-2]}",
+                    "%Y %b %d %H:%M",
+                )
+            except ValueError:
```

Comment:
I don't this `ValueError` handling is better way. 
As per my observation. when ftp `file/dir` created in back year then it will remove `time` and put `year`.